Sources/SwiftFormat/Utilities/FileIterator.swift

```diff
@@ -145,7 +145,7 @@ public struct FileIterator: Sequence, IteratorProtocol {
         // be displayed as relative paths. Otherwise, they will still be displayed as absolute
         // paths.
         let relativePath =
-          path.hasPrefix(workingDirectory.path)
+          path.hasPrefix(workingDirectory.path) && FileManager.default.currentDirectoryPath != "/"
```

Comment:
@compnerd What will this do on Windows? Probably not what's intended, I think?

Reply:
That was my concern, I don't know how Foundation will deal with that there.

Reply:
Actually it might be OK.

 https://github.com/swiftlang/swift-foundation/blob/ba000ae07c106149c2710bb9cb8c6993e20734b2/Sources/FoundationEssentials/FileManager/FileManager%2BDirectories.swift#L467

Reply:
`FileManager.default.currentDirectoryPath` will never be `/` on Windows because Windows always has a drive letter, eg. `C:\`. Checking if a URL is a root path on Windows is not quite trivial because you also need to account for networks shares eg. `\\your-server\blah` and the only good way of doing it issuing the `PathCchIsRoot` Windows API (eg. https://github.com/swiftlang/swift-tools-support-core/blob/5b130e04cc939373c4713b91704b0c47ceb36170/Sources/TSCBasic/Path.swift#L491). 

But maybe it’s not really an issue on Windows anyway, depending on how test output looks like. This is why I suggested making `workingDirectory` changeable from tests and then we can just check.

Reply:
I don't know that it is an issue on Windows either. Most people seem to have discovered it when using WASM. Are there Windows and Linux runners for CI so that we can see if this breaks anything? Otherwise I can install a toolchain on Windows here and check.

I can still make `workingDirectory` changeable from tests, but at this point I'm worried about the validity of the entire approach on Windows.

Reply:
I suppose I could also implement the `isRoot` property from the swift-tools-support-core as well in `FileIterator`. That way it would work on Windows as well.

Reply:
These tests do fail on Windows when trying to change the working directory. I'll update everything and get it working.

Reply:
If you don’t have a Windows machine to test this on, just write code that works to the best of you knowledge and some test cases for it. We have Windows PR testing (we just need to approve runs) and I’m happy to get the Windows code over the finish line if there are minor issues.

Reply:
Oh, that’s great. Let me know if you have any questions.

Reply:
Will do. Hoping to get back to this today.

Reply:
Let me know when you think the PR is ready for review again.